fiber/models.py

```diff
@@ -280,11 +280,15 @@ def resort():
                 resort()
 
 
+def images_directory(instance, filename):
+    return IMAGES_DIR
```

Comment:
https://docs.djangoproject.com/en/1.11/ref/models/fields/#django.db.models.FileField.upload_to

> upload_to may also be a callable, such as a function. This will be called to obtain the
> upload path, **including the filename.**